d2l-table-shared-styles.js

```diff
@@ -18,36 +18,6 @@ $_documentContainer.innerHTML = `<custom-style>
 			--d2l-table-row-border-color-selected: var(--d2l-color-celestine);
 			--d2l-table-row-background-color-selected: var(--d2l-color-celestine-plus-2);
 
-			--d2l-table-cell: {
-				display:table-cell;
-				vertical-align:middle;
-				padding: 0.5rem 1rem;
-				height: 41px; /* min-height to be 62px including border */
-			}
-			--d2l-table-light-cell: {
-				display: table-cell;
-				vertical-align: middle;
-				padding: 0.6rem;
-				height: 1.15rem; /* min-height to be 48px including border */
-			}
-
-			--d2l-table-header: {
-				color:var(--d2l-color-ferrite);
-				font-size:.7rem;
-				line-height: 0.9rem;
-				margin:1rem 0;
-				padding: 0.5rem 1rem;
-				height: 27px; /* min-height to be 48px including border */
-			}
-			--d2l-table-light-header: {
-				color: var(--d2l-color-ferrite);
-				font-size: 0.7rem;
-				font-weight: normal;
-				line-height: 0.9rem;
-				padding: 0.6rem;
-				height: 1.15rem; /* min-height to be 48px including border */
-			}
-
 			--d2l-table: {
```

Comment:
Is this just staying around to avoid an error in the https://search.d2l.dev/xref/Brightspace/insights-engagement-dashboard/components/table.js?r=2281dfce#188 case until it's removed then?

Reply:
Oops no I meant to remove it as well, which I now have. The usage in engagement-dashboard isn't actually doing anything (the table styles aren't loaded) it was just a copy-paste error. I'll be removing that usage shortly anyway but I'll need to make some other related changes to their code as they're using some other scroll-wrapper mixins.